frontend/asset_mapper.rst

```diff
@@ -188,6 +188,11 @@ to add any `npm package`_:
 
     $ php bin/console importmap:require bootstrap
 
+.. note::
+
+    If you're getting a 404 error, you can contact the package maintainer and ask them to
+    include `"main":` and `"module":` to the package's `package.json`.
+
```

Comment:
This cut the example in two. Could you group your two additions in a dedicated paragraph ? 

And a 404 could have many other reasons, i fear this could be misdirecting :|